examples/hello-spring-boot/README.md

```diff
@@ -0,0 +1,7 @@
+# Spring Boot Hello World with IntelliJ and Kubernetes
+
+This project is a very simple web application created with [the popular Spring Boot framework](https://spring.io/projects/spring-boot). It uses the [Jib Maven plugin](https://github.com/GoogleContainerTools/jib) to build a container image for the project, without needing to create a Dockerfile. It defines one HTTP endpoint ("/") and starts built-in Spring Boot web server on port 8080.
```

Comment:
very simple -> simple  (I think that's good enough)

Reply:
Cool, agreed.

Reply:
I'll update this in main README too.